res/css/voice-broadcast/atoms/_VoiceBroadcastRoomSubtitle.pcss

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_RoomTile .mx_RoomTile_titleContainer .mx_RoomTile_subtitle.mx_RoomTile_subtitle--voice-broadcast {
```

Comment:
non-blocking comment,

I'm not convinced we need to embed "voice broadcast" in the naming of this class.
 
I believe this should be part of the message preview CSS and have a name that describes its purpose rather than its function.
I can definitely see other cases where we'd want a leading icon then a message preview (threaded messages for example)

Reply:
D'accord. Currently calls are displayed in the same place. I will create an issue for a refactoring. Really would like to see more CSS framework than specialised cases all over the place as well.